### PR TITLE
Add any-to-any as a task to Hub

### PR DESCRIPTION
**Pull request by merveenoyan**
This PR adds any-to-any for tasks that have 2+ inputs to 2+ outputs like 4M, Chameleon, Lumina-mGPT etc
The icon looks like this: 

Screenshot 2024-08-21 at 16 15 10

### CODE REVIEW

**Review by osanseviero — COMMENTED** [review]
LGTM from a quick look and based in internal discussions

**Review by pcuenca — APPROVED** [review]
LGTM! What model candidates do we have so far? Chameleon, 4M, any others?

**Comment by merveenoyan** [comment]
@pcuenca and Lumina-mGPT but more are coming up

File: packages/widgets/src/lib/components/PipelineIcon/PipelineIcon.svelte [hunk 1]
```diff
@@ -96,6 +97,7 @@
 		"image-to-3d": IconImageTo3D,
 		"image-feature-extraction": IconImageFeatureExtraction,
 		"video-text-to-text": IconVideoTextToText,
+		"any-to-any": IconAnyToAny,
```
Comment (julien-c) [inline packages/widgets/src/lib/components/PipelineIcon/PipelineIcon.svelte:100]: i like the icon!

**Review by julien-c — APPROVED** [review]

**Comment by julien-c** [comment]
(IRL @gary149 was wondering about the naming of this task)

**Pull request merged**